packages/core/tests/js/vite/with-satisfies/output.ts

```diff
@@ -18,23 +18,24 @@ const config = defineConfig({
 	plugins: [
 		// all plugins
 		examples,
+
```

Comment:
These new empty lines are a bit annoying to me!
I'm not sure who is the cupid here... I'll check :D

Reply:
This is an `esrap` issue for sure. I think this only happens directly after comments, so this might be intentional. Without the new line you could technically read it as the command was supposed to be to be for all eleemnts. Which it might actually be in that case. In that case the comment should potentially be on `plugins` directly, but that's nothing we can change here.